tox.ini

```diff
@@ -13,17 +13,40 @@ basedeps = keyring
 deps = {[testenv]basedeps}
        django
 setenv =
-    PYTHONPATH=../google_appengine
     pypy: with_gmp=no
 commands = nosetests --ignore-files=test_appengine\.py {posargs}
 
-[testenv:cover]
+[coverbase]
 basepython = python2.7
 commands =
-    nosetests --with-xunit --with-xcoverage --cover-package=oauth2client --nocapture --cover-erase --cover-tests --cover-branches --ignore-files=test_appengine\.py --cover-min-percentage=60
+    nosetests \
+      --with-coverage \
+      --cover-package=oauth2client \
+      --cover-erase \
+      --cover-tests \
+      --cover-branches \
+      --ignore-files=test_appengine\.py
+    nosetests \
+      --with-coverage \
+      --cover-package=oauth2client \
+      --with-gae \
+      --cover-tests \
+      --cover-branches \
+      --gae-application=tests/data \
+      --gae-lib-root={env:GAE_PYTHONPATH} \
+      --logging-level=INFO \
+      tests/test_appengine.py
 deps = {[testenv]deps}
     coverage
-    nosexcover
+    nosegae
+
+[testenv:cover]
+basepython = {[coverbase]basepython}
+commands =
+    {[coverbase]commands}
+    coverage report -m --fail-under 80
```

Comment:
Since we have `{[testenv:cover]commands}` in `coveralls`, it will cause the failure we discussed before. You'll likely just want to swap it out from

```
commands =
    {[testenv:cover]commands}
    coveralls
```

to

```
commands =
    {[coverbase]commands}
    coverage report -m
    coveralls
```

----

Can you use `--show-missing` instead of `-m`?